pkg/cmd/cli/cmd/wrappers.go

```diff
@@ -230,7 +230,7 @@ func NewCmdCompletion(fullName string, f *clientcmd.Factory, out io.Writer) *cob
 		cmdHelpName = "openshift"
 	}
 
-	cmd := kcmd.NewCmdCompletion(f, out)
+	cmd := kcmd.NewCmdCompletion(f, out, "\n")
```

Comment:
I think what you did upstream is good and the right way to do it. For aesthetics, I would prefer to avoid a newline but I guess it's not easily doable.